### PR TITLE
docs: Update NodeClasses and NodePools for v1

### PR DESCRIPTION
**Pull request by chrisnegus**
**Description**: Updated NodeClasses and NodePools examples and associated descriptions in docs.

**Does this change impact docs?**
- [X ] Yes, PR includes docs updates 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/preview/concepts/nodepools.md [hunk 1]
```diff
@@ -167,6 +136,18 @@ spec:
   # to select. Higher weights indicate higher priority when comparing NodePools.
   # Specifying no weight is equivalent to specifying a weight of 0.
   weight: 10
+status:
```
Comment (engedaam) [inline website/content/en/preview/concepts/nodepools.md:139]: Should we add a status section that provides an explanation of each filed? Similar to how it's done for nodeclasses?

File: website/content/en/preview/concepts/nodeclasses.md [hunk 2]
```diff
@@ -159,12 +187,177 @@ status:
 
   # Generated instance profile name from "role"
   instanceProfile: "${CLUSTER_NAME}-0123456778901234567789"
+  conditions:
```
Comment (engedaam) [inline website/content/en/preview/concepts/nodeclasses.md:190]: Should we add a section for the condition fields?

File: website/content/en/preview/concepts/nodepools.md (hunk 1 above)
Comment (chrisnegus) [reply website/content/en/preview/concepts/nodepools.md:144]: I went through and made sure that all fields are now described.

File: website/content/en/preview/concepts/nodeclasses.md (hunk 2 above)
Comment (chrisnegus) [reply website/content/en/preview/concepts/nodeclasses.md:190]: I added descriptions of status.conditions and status.resources.

File: website/content/en/preview/concepts/nodepools.md [hunk 3]
```diff
@@ -331,158 +339,12 @@ spec:
...
-{{% /alert %}}
-
-{{% alert title="Pods Per Core on Bottlerocket" color="warning" %}}
-Bottlerocket AMIFamily currently does not support `podsPerCore` configuration. If a NodePool contains a `provider` or `providerRef` to a node template that will launch a Bottlerocket instance, the `podsPerCore` value will be ignored for scheduling and for configuring the kubelet.
-{{% /alert %}}
 
 ## spec.disruption
 
-You can configure Karpenter to disrupt Nodes through your NodePool in multiple ways. You can use `spec.disruption.consolidationPolicy`, `spec.disruption.consolidateAfter` or `spec.disruption.expireAfter`. Read [Disruption]({{<ref "disruption" >}}) for more.
+You can configure Karpenter to disrupt Nodes through your NodePool in multiple ways. You can use `spec.disruption.consolidationPolicy`, `spec.disruption.consolidateAfter` or `spec.disruption.expireAfter`.
```
Comment (engedaam) [inline website/content/en/preview/concepts/nodepools.md:345]: ExpireAfter is not in the disruption controls. Should be update once this PR gets rebased

**Comment by chrisnegus** [comment]
Replaced this PR with: https://github.com/aws/karpenter-provider-aws/pull/6549

File: website/content/en/preview/concepts/nodepools.md (hunk 3 above)
Comment (chrisnegus) [reply website/content/en/preview/concepts/nodepools.md:345]: Okay. Thanks. I'll take care of that in the other PR.

**Pull request closed without merging**